web/source/osk/browser/subkeyPopup.ts

```diff
@@ -211,5 +222,30 @@ namespace com.keyman.osk.browser {
         this.callout.parentNode.removeChild(this.callout);
       }
     }
+
+    updateTouch(touch: Touch) {
+      let x = touch.clientX;
+      let y = touch.clientY;
+
+      this.currentSelection = null;
+
+      for(let i=0; i < this.baseKey['subKeys'].length; i++) {
+        try {
+          let sk= this.element.childNodes[i].firstChild as KeyElement;
+          let x0 = dom.Utils.getAbsoluteX(sk); 
+          let y0 = dom.Utils.getAbsoluteY(sk);//-document.body.scrollTop;
+          
+          let x1=x0+sk.offsetWidth;
+          let y1=y0+sk.offsetHeight;
+
+          let onKey=(x > x0 && x < x1 && y > y0 && y < y1);
+          if(onKey) {
+            this.baseKey.key.highlight(false);
+            this.currentSelection = sk;
+          }
+          sk.key.highlight(onKey);
+        } catch(ex){}
+      }
+    }
```

Comment:
This (functionally) restores the following code removed by https://github.com/keymanapp/keyman/pull/5288/commits/063c326f5e0e088f85287129abbc20fbb3010820, as referenced in this PR's description:

https://github.com/keymanapp/keyman/blob/d68fb7bac3edc50fe16c296459f61ee4dce6ac90/web/source/osk/visualKeyboard.ts#L2353-L2364